glossary/IIFE.md

```diff
@@ -1,3 +1,17 @@
-# IIFE
+IIFE Immediately Invoked Function Expression is a function that gets called immediately after declaration.  Is most often used to create a scoping context.
```

Comment:
Leave the title up there please. :)
Also maybe *Immediately Invoked Function Expression* could be in parentheses?